Automattic/jetpack
# Update Authorize URL iframe to include affiliate code

Update the url on the authorization flow to include the affiliate code if the option is set along with the connection flow.d

Fixes #
N/A

#### Changes proposed in this Pull Request:

* The iframe for the in place connection flow currently does not include the affiliate code if the option is set. This updates the existing logic that builds the connection url and adds the affiliate code to also do the same for the authorization url.

#### Jetpack product discussion
https://wp.me/pbtFPC-E2-p2


#### Testing instructions:
* Pull down the branch on a site that does not yet have Jetpack connected. 
* Add the option `jetpack_affiliate_code` to the site options.
* When attempting to connect the site verify that the iframe url associated with the final connection step at "Connecting as {user}" also includes `aff=jetpack_affiliate_code` in the url query args.

#### Proposed changelog entry for your changes:

* Update the authorize url in the connection iframe in the final connection step to include the affiliate code similar to the existing connection url.
